src/contracts/fund/trading/Trading.test.ts

```diff
@@ -124,15 +125,14 @@ describe('Trading', () => {
         new BigNumber(0),
         new BigNumber(0),
       ],
-      identifier: 'anything',
+      identifier: padLeft('0x79705be7', 64),
```

Comment:
What's this :-P

Reply:
Identifier now needs to be 64 chars (32 hex chars).